app/src/main/java/org/oppia/android/app/topic/TopicViewModel.kt

```diff
@@ -20,6 +20,7 @@ class TopicViewModel @Inject constructor(
 ) : ObservableViewModel() {
   private var internalProfileId: Int = -1
   private lateinit var topicId: String
+  var enableMyDownloads: Boolean = false
```

Comment:
Why not inject the flag, instead? Ditto for other injectable view models. For non-injectable ones, prefer passing in the flag as a val to the constructor.

This value cannot change at runtime, so it ought to be a 'val' everywhere.

Reply:
fixed